apis/python/src/tiledbsoma/soma_array.cc

```diff
@@ -199,9 +199,7 @@ void load_soma_array(py::module &m) {
             auto pa_schema_import = pa.attr("Schema").attr("_import_from_c");
             return pa_schema_import(py::capsule(reader.arrow_schema().get()));
         })
-        .def("config", [](SOMAArray& reader) -> py::dict {
-            return py::cast(reader.config());
-        })
+        .def("ctx", &SOMAArray::ctx)
```

Comment:
The Python bindings use the name `context`, rather than `ctx` (or in TileDB-PY, `Ctx`). Is there a reason to introduce a third option (`ctx`) rather than stick with the SOMA name `context`?

(this is a nit: if there is a reason, I have no objection - just spent a few minutes trying to understand if they were different concepts or the same, just renamed)

Reply:
No there isn't a reason for using `ctx`. I'll make them all `context`.